packages/wdio-cli/src/constants.js

```diff
@@ -114,6 +115,45 @@ export const QUESTIONNAIRE = [{
     message: 'What is the port on which that service is running?',
     default: '80',
     when: /* istanbul ignore next */ (answers) => answers.backend.indexOf('different service') > -1
+}, {
+    type: 'input',
+    name: 'exp_env_access_key',
+    message: 'Access key from Experitest Cloud',
+    default: 'EXPERITEST_ACCESS_KEY',
+    when: /* istanbul ignore next */ (answers) => answers.backend === 'In the Experitest cloud'
+}, {
+    type: 'input',
+    name: 'exp_env_hostname',
+    message: 'Environment variable for cloud url',
+    default: 'example.experitest.com',
+    when: /* istanbul ignore next */ (answers) => answers.backend === 'In the Experitest cloud'
+}, {
+    type: 'list',
+    name: 'exp_env_port',
+    message: 'Choose a port for environment variable',
```

Comment:
Let's only allow SSL ports (443), I don't think there is a scenario where someone prefers an HTTP connection

Reply:
Experitest offers an on-premise (non-SASS) selenium hub for our customers. 
As such our customers are those who decide which protocol or port to use.
We cannot limit them to port 443.